ci-operator/step-registry/baremetal/lab/pre/dhcp-conf/baremetal-lab-pre-dhcp-conf-commands.sh

```diff
@@ -22,6 +22,9 @@ fi
 
 echo "Generating the DHCP/PXE config..."
 
+# DHCP unique identifier (DUID)
+DUID="00:03:00:01"
```

Comment:
Is this the same for any possible server?

Reply:
Yes, as long as it uses Ethernet:

- DUID type—The value 0x0003 indicates that the DUID type is DUID-LL.
- Hardware type—Ethernet as the hardware type with the value of 0x0001.
- Link layer address—Its value MAC address